stable_baselines3/common/type_aliases.py

```diff
@@ -69,3 +70,26 @@ class TrainFrequencyUnit(Enum):
 class TrainFreq(NamedTuple):
     frequency: int
     unit: TrainFrequencyUnit  # either "step" or "episode"
+
+
+class PolicyPredictor(Protocol):
```

Comment:
nice, I guess protocol is the equivalent of "Trait" in Rust?
(shared behavior)

Reply:
Yep, it's similar! Defined in https://peps.python.org/pep-0544/#protocol-members